pyg/Pygemony.py

```diff
@@ -45,7 +45,7 @@ def _sanitize_todo_line(self, lines):
     @staticmethod
     def hash_todo(todo_content, file_name):
         m = hashlib.md5()
-        m.update('{}-{}'.format(todo_content, file_name))
+        m.update('{0}-{1}'.format(todo_content, file_name))
```

Comment:
Its interesting that the tests passed under python 2.6 before, because they shouldn't have :)